includes/class-wc-gateway-ppec-cart-handler.php

```diff
@@ -509,13 +509,14 @@ public function enqueue_scripts() {
 	 * Creates a customer session if one is not already active.
 	 */
 	public function ensure_session() {
-		$frontend = ( ! is_admin() || defined( 'DOING_AJAX' ) ) && ! defined( 'DOING_CRON' ) && ! defined( 'REST_REQUEST' );
+		$is_rest_request = is_callable( WC(), 'is_rest_api_request' ) ? WC()->is_rest_api_request() : defined( 'REST_REQUEST' );
```

Comment:
Do we need to update how we check for a REST API request, or is this for testing purposes?